posts/how-to-commit-code-to-a-git-repository-with-visual-studio-code.md

```diff
@@ -0,0 +1,172 @@
+    <figcaption>Location of the <strong>Source Control</strong> icon in the Activity Bar.</figcaption>
+</figure>
+
+Click on the icon to open the **Source Control** panel. This panel provides an interface to view the current changes in your working directory and to create new commits.
+
+<figure className="small">
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-panel.png" alt="The Source Control panel displays some changes in the repository." />
+    <figcaption>The <strong>Source Control</strong> panel displays some changes in the repository.</figcaption>
+</figure>
+
+The **Changes** section of the panel lists all the files in the working directory that have been changed. The total number of changed files is also reflected on the **Source Control** icon’s badge. In the example in the above screenshot, the badge indicates that there are three changed files, and those three files are listed in the **Changes** section.
+
+By selecting one of the files in the list, we can view the changes to that file. VS Code displays the changes as a **diff view** in a new editor tab. For example, if we select the `README.md` file, we can browse the changes as displayed in the screenshot below.
+
+<figure>
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-diff-view.png" alt="An example of a diff view of the changes made to README.md." />
+    <figcaption>An example of a diff view of the changes made to <code>README.md</code>.</figcaption>
+</figure>
+
+To prepare a commit, we will select changes to add to the staging area.
+
+## Step 2 — Adding changes to the staging area
+
+The **staging area** in Git is the collection of changes that will be included in the next commit. The next step is to add some changes to the staging area.
+
+### Staging individual files
+
+Each file in the **Changes** list represents a change in the repository’s working directory that has not yet been staged and is available for staging.
+
+The badge next to each file indicates the type of change:
+
+- The `M` badge indicates that a file is modified. This means that it exists in the repository and has changed since the last commit.
+- The `U` badge represents an untracked file, which means it doesn’t exist in the repository and was newly created since the last commit.
+- The `D` badge means that the file was deleted.
+
+Each file has three action buttons. You can reveal these buttons by hovering or clicking on a file in the list.
+
+<figure className="small" >
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-stage-file.png" alt="Actions for each file in the Changes section." />
+    <figcaption>Actions for each file in the <strong>Changes</strong> section.</figcaption>
+</figure>
+
+From left to right, the action buttons are:
+
+- **Open File**, which opens the file in a new editor tab for editing.
```

Comment:
These need to be consistent, the first one has a comma, the second two have periods. I would do as the first or if you want to follow a format similar to the second two, use colons instead